File: src/main/java/project/rpg/skill/base/Operable.java
package project.rpg.skill.base;

import org.bukkit.entity.Player;
import org.bukkit.event.block.Action;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

public interface Operable {

    void onEnable(@NotNull Player player, @NotNull Action action);
}

File: src/main/java/project/rpg/skills/base/Operable.kt
package project.rpg.skills.base

import org.bukkit.entity.Player
import org.bukkit.event.block.Action

interface Operable {

    /**
     * @param player 작동시킨 플레이어
     * @param action 작동시킬 때 동작
     */
    fun onEnable(player: Player, action: Action?)
}